### DISCUSSION TITLE
How to reset text-encoder training

### DISCUSSION THREAD

**Discussion by OEvgeny** [TheLastBen/fast-stable-diffusion#873] [Q&A] [answered]
I have a v2 model which works good after 6000 steps, but textencoder was trained for 4200 steps which is way more then needed, so it puts my training subject to every prompt I try.

Is there a way I can re-train text encoder from scratch for the model I have?

**Comment by TheLastBen** [comment, marked as answer]
I'm working on adding that feature, keep the model, once the feature gets added, you can fix it
